test/action_controller/serialization_scope_name_test.rb

```diff
@@ -29,12 +29,12 @@ def render_new_user
 
   def test_default_scope_name
     get :render_new_user
-    assert_equal '{"users":{"admin?":false}}', @response.body
+    assert_equal '{"data":{"admin?":false,"id":"1","type":"test_users"}}', @response.body
```

Comment:
@kurko  I've refactored the test code a little bit, the output of type is now `"users"` as expected